.github/dependabot.yml

```diff
@@ -41,3 +41,20 @@ updates:
       github-actions:
         patterns:
           - "*"
+  # To update all the images in the operatorconfig and tests/e2e/testfiles directory
+  - package-ecosystem: "docker"
+    directories:
+      # Update the version to latest post release to stay up to date
+      - "/operatorconfig/driverconfig/powerflex/v2.15.0/**"
+      - "/operatorconfig/driverconfig/powermax/v2.15.0/**"
+      - "/operatorconfig/driverconfig/powerscale/v2.15.0/**"
+      - "/operatorconfig/driverconfig/powerstore/v2.15.0/**"
+      - "/operatorconfig/driverconfig/unity/v2.15.0/**"
```

Comment:
will we have to update the latest version in the coming release?

Reply:
yes, there is an option to automate that as well, but again it creates another PR. For now, we can stick to this and on our new release we can try to add that automation as well.

Reply:
Ok